esp/src/eclwatch/ESPWorkunit.js

```diff
@@ -312,7 +312,13 @@ define([
             WsWorkunits.WUUpdate({
                 request: request,
                 load: function (response) {
-                    context.updateData(response.WUUpdateResponse.Workunit);
+                    if (response.Exceptions) {
+                        arrayUtil.forEach(response.Exceptions.Exception, function (item, idx) {
+                            context.updateData(response.Exceptions.Exception[idx].Message);
```

Comment:
The message here is ""GMT: Invalid cluster name: ClusterName" otherwise we get the long dojo.js toaster message.